### PR TITLE
25 updating page preserving

### PR DESCRIPTION
**Pull request by LizGuzRam**

### CODE REVIEW

File: pages/data_life_cycle/preserving.md [hunk 1]
```diff
@@ -69,6 +69,13 @@ When preparing data for preservation several requirements need to be fulfilled.
 - Ensure that a consistent naming scheme is used.
 - Use standard, open source, file formats instead of proprietary ones.
 
+<!-- ERIM specific info (begin)-->
+
+At EUR we recommend researchers to use [EUR Yoda Vault](https://my.eur.nl/en/eur-employee/data-archiving-eur-yoda-vault) in order for reseachers to archive and preserve their data for verification purposes.
```
Comment (nsunami) [inline pages/data_life_cycle/preserving.md:74]: I'm just suggesting this to change to the public URL 👍 

```suggestion
At EUR we recommend researchers to use [EUR Yoda Vault](https://eur.nl/en/data-archiving-eur-yoda-vault) in order for reseachers to archive and preserve their data for verification purposes. 
```

**Review by nsunami — CHANGES_REQUESTED** [review]
I have one small suggestion to change the MyEUR URL to the public URL. Otherwise, it looks good to me!

File: pages/data_life_cycle/preserving.md (hunk 1 above)
Comment (LizGuzRam) [reply pages/data_life_cycle/preserving.md:74]: Yeah way better to use the public links! :)

**Review by nsunami — APPROVED** [review]
Looks good!

**Pull request merged**